lib/tbd/geo.rb

```diff
@@ -251,7 +251,7 @@ def faces(s = {}, e = {})
   #
   # @return [Topolys::Vector3D] true normal vector of s
   # @return [nil] if invalid input (see logs)
-  def trueNormal(s = nil, r = 0)
+  def truNormal(s = nil, r = 0)
```

Comment:
Distinguishes OSut's `trueNormal` from TBD's internal `truNormal`. The obvious way around this is proper namespaces, but this will do for now.